config/validate/validate.go

```diff
@@ -42,6 +42,8 @@ func Validate(userdataBytes []byte) (Report, error) {
 		return Report{}, nil
 	case config.IsIgnitionConfig(string(userdataBytes)):
 		return Report{}, nil
+	case config.IsMimeMultiPart(string(userdataBytes)):
+		return Report{}, nil
```

Comment:
The error message for this validate should also mention that multipart mime is a supported format.